### ISSUE TITLE
Add functionality to re-ask user for an option to the Scribe-Data interactive mode

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#209]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The Scribe-Data interactive mode can be accessed in the following way:

```bash
scribe-data get -i
```

With this the user is prompted to select options that would be appropriate to get the data that they need. One thing that's missing is functionality to re-ask the user for a response if they did not provide one. If the user provides an invalid response, then the program should error, but if the user just hits enter then they should be asked the same question again - i.e.:

```bash
$ scribe-data get -i
Welcome to Scribe-Data interactive mode!

Language options:
1. English
2. French
3. German
...

Please enter the languages to get data for, their numbers or (a) for all languages:

# User hits enter, and what should follow is...

Language options:
1. English
2. French
3. German
...

Please enter the languages to get data for, their numbers or (a) for all languages:
```

### Contribution

Happy to support this with a review and answer any questions a contributor might have!

**Comment by KesharwaniArpita** [comment]
@andrewtavis I have an idea for this. I think if we update the interactive.py with the following changes we will be able to add the functionality to re-ask the user for a response if they haven't provided one:
1. Modify the `get_selection()` function to re-ask for input if no valid input is provided.
2. Add a loop in the `select_languages()` and `select_data_types()` functions to handle re-asking the question.
3. Make the output options prompt also re-ask the user in case of an empty response.

**Comment by andrewtavis** [comment]
Sounds great, @KesharwaniArpita! Let's finish the other issue and then we can move on to this one :)

**Comment by omoladd** [comment]
Hi @andrewtavis  , I'm interested in this. Can you assign this to me?

**Comment by KesharwaniArpita** [comment]
Hi @omoladd, I was very curious about how to solve this issue and tried to tweak the existing functions a little bit in the day. Since you also seem interested would you like to take a look into the PR and see if it can be improved your suggestion would mean a lot.

**Comment by omoladd** [comment]
@KesharwaniArpita, I didn’t realize you already responded. 
I opened a PR too; https://github.com/scribe-org/Scribe-Data/pull/239

**Comment by KesharwaniArpita** [comment]
Great. Would love to colab!!!

**Comment by omoladd** [comment]
Sure! We can communicate better on matrix.

**Comment by andrewtavis** [comment]
Closed by #268 for the new interactive mode interface :) Thanks, @axif0, and also thanks @KesharwaniArpita and @omoladd!
